tools/tests/form-control/test.js

```diff
@@ -427,21 +450,80 @@ function setFormsTestCases() {
 	}
 }
 
+function setFormTestCasesWithInteraction() {
+	for (const formData of forms) {
+		if (!formData.submitDataInteraction) continue; // Skipping PREVENT_SUBMIT_FORM
```

Comment:
I've reused the same `forms` array that is used when running the tests from `setFormsTestCases()` and it is present because it is used there. I wanted to avoid creating another very similar array.